nvme_gw_server.py

```diff
@@ -528,74 +532,96 @@ def nvmf_subsystem_remove_host(self, request, context=None):
     def nvmf_subsystem_add_listener(self, request, context=None):
         """Adds a listener at the given TCP/IP address for the given subsystem."""
         self.logger.info({
-            f"Adding {request.trtype} listener at {request.traddr} : {request.trsvcid} for {request.nqn}"
+            f"Adding {request.gateway} {request.trtype} listener at {request.traddr} : {request.trsvcid} for {request.nqn}"
```

Comment:
Get rid of spaces between the address and the port:
```suggestion
            f"Adding {request.gateway} {request.trtype} listener at {request.traddr}:{request.trsvcid} for {request.nqn}"
```